test/elli_tests.erl

```diff
@@ -735,7 +736,8 @@ register_test() ->
     ok.
 
 invalid_callback_test() ->
-    case catch elli:start_link([{callback, elli}]) of
-        E ->
-            ?assertMatch(invalid_callback, E)
+    try
+        elli:start_link([{callback, elli}])
+    catch throw:E ->
```

Comment:
Doesn't this change the functionality? Now it is a crash if the exception is anything but `throw`, while before `catch` handled `exit` and `error` as well.

Reply:
This is a test. I thought it should be more specific, since we know what we're testing, but I can go back on that assumption.